pkg/internal/checkup/checkup.go

```diff
@@ -390,6 +402,61 @@ func (c *Checkup) checkDefaultStorageClass(scs *storagev1.StorageClassList, errS
 	}
 }
 
+func (c *Checkup) checkPVCBound(ctx context.Context, errStr *string) error {
+	log.Print("checkPVCCreation")
+
+	pvc := &corev1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: pvcName,
+			OwnerReferences: []metav1.OwnerReference{{
+				APIVersion: "v1",
+				Kind:       "Pod",
+				Name:       c.checkupConfig.PodName,
+				UID:        types.UID(c.checkupConfig.PodUID),
+			}},
+		},
+		Spec: corev1.PersistentVolumeClaimSpec{
+			AccessModes: []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
+			Resources: corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceStorage: resource.MustParse("10Mi"),
+				},
+			},
+		},
+	}
+
+	if sc := c.checkupConfig.StorageClass; sc != "" {
+		log.Printf("PVC storage class %q", sc)
+		pvc.Spec.StorageClassName = &sc
+	}
+
+	if _, err := c.client.CreatePersistentVolumeClaim(ctx, c.namespace, pvc); err != nil {
+		return err
+	}
+
+	c.waitForPVCBound(ctx, &c.results.PVCBound, errStr)
+
+	return c.client.DeletePersistentVolumeClaim(ctx, c.namespace, pvcName)
+}
+
+func (c *Checkup) waitForPVCBound(ctx context.Context, result, errStr *string) {
+	conditionFn := func(ctx context.Context) (bool, error) {
+		pvc, err := c.client.GetPersistentVolumeClaim(ctx, c.namespace, pvcName)
+		return pvc.Status.Phase == corev1.ClaimBound, err
+	}
+
+	log.Printf("Waiting for PVC %q bound", pvcName)
+	if err := wait.PollImmediateWithContext(ctx, pollInterval, time.Minute, conditionFn); err != nil {
```

Comment:
So in case the provisioner binding mode is WFFC we are returning error? why is this the desired behavior?

Reply:
so we better create DV with `cdi.kubevirt.io/storage.bind.immediate.requested: "true"` instead?

Reply:
Im not sure what we want to test exactly.. so I guess is depends

Reply:
all we want to check is that a PVC of the configured/default SC can be successfully bound

Reply:
ok then I guess yes we should use the bind annotation